t/mws_range/mws_range.xml

```diff
@@ -0,0 +1,47 @@
+        <mrow xml:id="p1.1.m1.1.1" xref="p1.1.m1.1.1.cmml"><mo mathcolor="red" xml:id="p1.1.m1.1.1a" xref="p1.1.m1.1.1.cmml">[</mo>1...5<mo mathcolor="red" xml:id="p1.1.m1.1.1b" xref="p1.1.m1.1.1.cmml">]</mo></mrow>
+        <mo xml:id="p1.1.m1.1.11.1.1" xref="p1.1.m1.1.11.1.1.cmml">⁢</mo>
+        <mrow xml:id="p1.1.m1.1.2" xref="p1.1.m1.1.2.cmml"><mo mathcolor="red" xml:id="p1.1.m1.1.2a" xref="p1.1.m1.1.2.cmml">[</mo>...5<mo mathcolor="red" xml:id="p1.1.m1.1.2b" xref="p1.1.m1.1.2.cmml">]</mo></mrow>
+        <mo xml:id="p1.1.m1.1.11.1.1a" xref="p1.1.m1.1.11.1.1.cmml">⁢</mo>
+        <mrow xml:id="p1.1.m1.1.3" xref="p1.1.m1.1.3.cmml"><mo mathcolor="red" xml:id="p1.1.m1.1.3a" xref="p1.1.m1.1.3.cmml">[</mo>1...<mo mathcolor="red" xml:id="p1.1.m1.1.3b" xref="p1.1.m1.1.3.cmml">]</mo></mrow>
+        <mo xml:id="p1.1.m1.1.11.1.1b" xref="p1.1.m1.1.11.1.1.cmml">⁢</mo>
+        <mrow xml:id="p1.1.m1.1.4" xref="p1.1.m1.1.4.cmml"><mo mathcolor="red" xml:id="p1.1.m1.1.4a" xref="p1.1.m1.1.4.cmml">[</mo>...<mo mathcolor="red" xml:id="p1.1.m1.1.4b" xref="p1.1.m1.1.4.cmml">]</mo></mrow>
+        <mo xml:id="p1.1.m1.1.11.1.1c" xref="p1.1.m1.1.11.1.1.cmml">⁢</mo>
+        <mrow xml:id="p1.1.m1.1.5" xref="p1.1.m1.1.5.cmml"><mo mathcolor="red" xml:id="p1.1.m1.1.5a" xref="p1.1.m1.1.5.cmml">[</mo>1...5<mo mathcolor="red" xml:id="p1.1.m1.1.5b" xref="p1.1.m1.1.5.cmml">]</mo></mrow>
+      </mrow>
+      <mo xml:id="p1.1.m1.1.6" xref="p1.1.m1.1.6.cmml">+</mo>
+      <mrow xml:id="p1.1.m1.1.11.2" xref="p1.1.m1.1.11.2.cmml">
+        <mi mathcolor="red" xml:id="p1.1.m1.1.7" xref="p1.1.m1.1.7.cmml">x</mi>
+        <mo xml:id="p1.1.m1.1.11.2.1" xref="p1.1.m1.1.11.2.1.cmml">⁢</mo>
+        <mrow xml:id="p1.1.m1.1.8" xref="p1.1.m1.1.8.cmml"><mo mathcolor="red" xml:id="p1.1.m1.1.8a" xref="p1.1.m1.1.8.cmml">[</mo>9.00...11.00<mo mathcolor="red" xml:id="p1.1.m1.1.8b" xref="p1.1.m1.1.8.cmml">]</mo></mrow>
+        <mo xml:id="p1.1.m1.1.11.2.1a" xref="p1.1.m1.1.11.2.1.cmml">⁢</mo>
+        <mrow xml:id="p1.1.m1.1.9" xref="p1.1.m1.1.9.cmml"><mo mathcolor="red" xml:id="p1.1.m1.1.9a" xref="p1.1.m1.1.9.cmml">[</mo>47.50...52.50<mo mathcolor="red" xml:id="p1.1.m1.1.9b" xref="p1.1.m1.1.9.cmml">]</mo></mrow>
+        <mo xml:id="p1.1.m1.1.11.2.1b" xref="p1.1.m1.1.11.2.1.cmml">⁢</mo>
+        <mrow xml:id="p1.1.m1.1.10" xref="p1.1.m1.1.10.cmml"><mo mathcolor="red" xml:id="p1.1.m1.1.10a" xref="p1.1.m1.1.10.cmml">[</mo>-1.00...1.00<mo mathcolor="red" xml:id="p1.1.m1.1.10b" xref="p1.1.m1.1.10.cmml">]</mo></mrow>
+      </mrow>
+    </mrow>
+    <annotation-xml xmlns:m="http://www.w3.org/1998/Math/MathML" xmlns:mws="http://search.mathweb.org/ns" encoding="MWS-Query">
+      <apply xml:id="p1.1.m1.1.11.cmml" xref="p1.1.m1.1.11">
+        <plus xml:id="p1.1.m1.1.6.cmml" xref="p1.1.m1.1.6"/>
+        <apply xml:id="p1.1.m1.1.11.1.cmml" xref="p1.1.m1.1.11.1">
+          <times xml:id="p1.1.m1.1.11.1.1.cmml" xref="p1.1.m1.1.11.1.1"/>
```

Comment:
Maybe this is just pragmatic for testing, but it's one strange content MathML tree - you multiply 5 ranges together? Wouldn't you prefer to have five separate formulas, each containing one range?  Just a thought.